heartbeat/rabbitmq-cluster.in

```diff
@@ -488,7 +488,11 @@ rmq_try_start() {
 
 	rmq_stop
 	rmq_wipe_data
-	rmq_forget_cluster_node_remotely "$join_list" "$local_rmq_node"
+        if [ -z "$local_rmq_node" ]; then
```

Comment:
You need to follow the use tab for the indents to follow the indentation of the code.

Reply:
Thanks for the catch. I fixed it.